.gitignore

```diff
@@ -1,4 +1,7 @@
-/node_modules/
-/dist
+.idea/
```

Comment:
This can be put in your local gitignore? 

e.g. ` cat ~/.gitignore
#VSCODE
.vscode/`

Reply:
Doesn't that mean that anyone should have it local `.gitignore`? But this way it will be ignored for all devs.

Reply:
We don't have standard set of tools, so every dev should probably update their own local `.gitignore` with the stuff they don't want to commit (e.g. `.DS_store` should be in local as well IMO).